apsconnectcli/apsconnect.py

```diff
@@ -766,6 +776,47 @@ def _get_properties(app_schema_path):
         return app_properties
 
 
+def _check_binding(check_config, get_config_info):
+    state_not_initiated = '\tNot initiated;'
+    state_is_ready = '\thost: {};\n\tuser: {};'
+    state_config_corrupted = '\tConfig file is corrupted: {};'
+
+    if not check_config():
+        return state_not_initiated
+
+    try:
+        info = get_config_info()
+    except Exception as e:
+        return state_config_corrupted.format(e)
+
+    if info == NULL_CFG_INFO:
+        return state_config_corrupted.format('binding attributes are not assigned')
```

Comment:
Use double quotes ("") when writing literals that would be output via `print` or similar. This is useful in case of future localisation.